src/app/charts/core/cminfo.yaml

```diff
@@ -6,3 +6,7 @@ alfresco:
 s3:
   aliases:
     - minio
+
+box:
+  aliases:
+    - box
```

Comment:
Aliases are only needed if the alias is different than the main dialect's name (case-insensitive)